cmd/snap/cmd_debug_change_timins.go

```diff
@@ -0,0 +1,89 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2019 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package main
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/jessevdk/go-flags"
+)
+
+type cmdChangeTimings struct {
+	changeIDMixin
+}
+
+func init() {
+	addDebugCommand("change-timings",
+		"Get the timings of the tasks of a change",
+		"Details about tte time each task runs.",
```

Comment:
s/tte/the/

Reply:
`The change-timings command displays ...`

Also, -1 to `change-timings`, especially because unless you know what it does you might think it's about changing the timings of something, but also because the command to get the tasks of a change is `snap tasks` (with `change` an unlisted legacy alias). `snap debug tasks-timings` sounds weird though. Why not `snap debug tasks`?

Reply:
"snap debug tasks" is a bit too vague vs the purpose. We might could it  just "timings" and extend its remit later to encompass ensure loop timings as well?